data/documents_big.json

```diff
@@ -31042,7 +31096,7 @@
       "Inuits",
       "D\u00e9veloppement durable",
       "Tourisme culturel",
-      "Ressources naturelles",
+      "Resources naturelles",
```

Comment:
In French, it's ressources, but resource in English. I think Ressources was correct here.

Reply:
Yes, corrected ....

Reply:
Seems that "resources naturelles" is wrong again.
Cf. https://cnrtl.fr/definition/ressource